### ISSUE TITLE
Dynamic Window dim(X) error

### ISSUE THREAD

**Issue by GACGAMA** [xihaoli/STAARpipeline-Tutorial#55]
Hello Xihao!

At the Dynamic Window step, i'm getting the following error on some jobs (53 for example):

```
Attaching package: ‘SCANG’

The following object is masked from ‘package:STAAR’:

    example

Error in apply(emthr_SCANG_O, 2, max) : 
  dim(X) must have a positive length
Calls: Dynamic_Window_SCANG -> apply
Execution halted
```

This results in no _53.Rdata output for example.  And then, when summarizing, I get the error that this slice is missing, but the files are NOT created. Im not sure if this error is innocuous

```
In readChar(con, 5L, useBytes = TRUE) :
  cannot open compressed file '/scratch4/STAAR/stepB/7/Disease_Controls_Dynamic_Window_53.Rdata', probabl  e reason 'No such file or directory'
```

**Comment by xihaoli** [comment]
Hi @GACGAMA,

I am not sure if this is caused by the fact that you have a small sample size. See [#40] for an example.

Best,
Xihao

**Comment by GACGAMA** [comment]
Running with double the samples worked

**Comment by jingydz** [comment]
Hello, I have looked into issue 40, and I am working with nearly 2,000 samples, but I have encountered the same error. 

Moreover, I have set the MAF (Minor Allele Frequency) to 0.05, which, however, does not seem to impact the dynamic_window. I observed that the MAF value is only configured with a rare_maf_cutoff=0.05 setting during the gene-centric script analysis. 

You mentioned running the analysis with double the number of samples; how is it possible to effectively double the sample size?
